lms/djangoapps/instructor/settings/common.py

```diff
@@ -10,35 +10,85 @@ def plugin_settings(settings):
-        # Display the 'Analytics' tab in the instructor dashboard for CCX courses.
-        # Note: This has no effect unless ANALYTICS_DASHBOARD_URL is already set,
-        #       because without that setting, the tab does not show up for any courses.
+        # .. toggle_name: FEATURES['ENABLE_CCX_ANALYTICS_DASHBOARD_URL']
+        # .. toggle_implementation: DjangoSetting
+        # .. toggle_default: False
+        # .. toggle_description: Display the 'Analytics' tab in the instructor dashboard for CCX courses.
+        #   Note: This has no effect unless ANALYTICS_DASHBOARD_URL is already set, because without that
+        #   setting, the tab does not show up for any courses.
+        # .. toggle_use_cases: opt_in
+        # .. toggle_creation_date: 2016-10-07
+        # .. toggle_tickets: https://github.com/edx/edx-platform/pull/13196
         'ENABLE_CCX_ANALYTICS_DASHBOARD_URL': False,
 
-        # Disable instructor dash buttons for downloading course data
-        # when enrollment exceeds this number
+        # .. setting_name: FEATURES['MAX_ENROLLMENT_INSTR_BUTTONS']
+        # .. setting_default: 200
+        # .. setting_description: Disable instructor dashboard buttons for downloading course data
+        #   when enrollment exceeds this number. The number indicates the maximum allowed enrollments
+        #   for the course to be considered "small". Courses exceeding the upper limit of "small"
+        #   courses will have disabled buttons at the instructor dashboard.
         'MAX_ENROLLMENT_INSTR_BUTTONS': 200,
 
-        # Grade calculation started from the instructor dashboard will write grades
-        # CSV files to the configured storage backend and give links for downloads.
+        # .. toggle_name: FEATURES['ENABLE_GRADE_DOWNLOADS']
+        # .. toggle_implementation: DjangoSetting
+        # .. toggle_default: False
+        # .. toggle_description: Enable grade CSV downloads from the instructor dashboard. Grade
+        #   calculation started from the instructor dashboard will write grades CSV files to the
+        #   configured storage backend and give links for downloads.
+        # .. toggle_use_cases: opt_in
+        # .. toggle_creation_date: 2016-07-06
+        # .. toggle_tickets: https://github.com/edx/edx-platform/pull/11286
         'ENABLE_GRADE_DOWNLOADS': False,
 
-        # Give course staff unrestricted access to grade downloads (if set to False,
-        # only edX superusers can perform the downloads)
+        # .. toggle_name: FEATURES['ALLOW_COURSE_STAFF_GRADE_DOWNLOADS']
+        # .. toggle_implementation: DjangoSetting
+        # .. toggle_default: False
+        # .. toggle_description: Enable to give course staff unrestricted access to grade downloads;
+        #   if set to False, only edX superusers can perform the downloads.
+        # .. toggle_use_cases: opt_in
+        # .. toggle_creation_date: 2018-03-26
+        # .. toggle_tickets: https://github.com/edx/edx-platform/pull/17385
```

Comment:
This PR seems to just be a refactor.  I'd drop this optional annotation, unless you find a different PR and you think it is important.  Thanks.